test/pulp3_api_test.rb

```diff
@@ -14,6 +14,7 @@ def app
   end
 
   def test_returns_pulp_status_on_200
+    PulpProxy::Pulp3Plugin.load_test_settings({})
```

Comment:
Looks like it previously depended on other tests to initialize it. Should fix the tests now.